### PR TITLE
Making resources (mapper resources) part of the ETLBatchConfig

### PR DESCRIPTION
**Pull request by gokulavasan**
Instead of having it as part of the BatchContext.

Bamboo : http://builds.cask.co/browse/CDAP-DT25

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/templates/etl/batch/ETLMapReduce.java [hunk 1]
```diff
@@ -77,6 +74,7 @@ public void beforeSubmit(MapReduceContext context) throws Exception {
     prepareSource(context, config.getSource());
     prepareSink(context, config.getSink());
 
+    context.setMapperResources(config.getResources());
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/templates/etl/batch/ETLMapReduce.java]: resources can be null right? if so should check before setting.

**Comment by albertshau** [comment]
one comment, otherwise lgtm

**Comment by gokulavasan** [comment]
@albertshau Addressed comments. Please review.

**Comment by gokulavasan** [comment]
Build passed : http://builds.cask.co/browse/CDAP-DT25-5

**Comment by albertshau** [comment]
lgtm

**Pull request merged**